submissions/benjaminKamena/phpDataase/ordersQueryDb.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+include('connected.php');
+$id =  $_GET['id'];
+try {
+	$sql= "SELECT  order_items.unit_price, order_items.quantity, order_items.total_amount, 
+	orders.order_created_at, orders.id, products.product_name, addresses.street_address, addresses.city, 
+	addresses.state, addresses.country
+    FROM order_items 
+    LEFT JOIN orders ON orders.id = order_items.order_id
+    LEFT JOIN products ON products.id = order_items.product_id
+    LEFT JOIN addresses ON orders.customer_orders = addresses.customer_id
+    WHERE orders.customer_orders = '$id'";
+
+    $stmt = $this->conn()->query($sql);
+    $orders = $stmt->fetchAll();
+	setcookie("view", serialize($orders), time()+7200, "/");
+	
+	$orders = isset($_COOKIE['view']) ? unserialize($_COOKIE['view']) : $view->getAllCustomers($id);
```

Comment:
The way you wrote your code, anytime there is a page reload, the `SELECT` query will run even when cookie is set. This isn't the idea. What we want is that the `SELECT` query only run when cookie has expired.

A better implementation is to separate your query from your view (I,e what is seen on the browser). You can put the query is something like a function. Then only invoke or call this function when cookie is not set